tests/add/test_add_core.py

```diff
@@ -63,9 +63,12 @@ def test_add_two_versions_of_bundle(store, bundle_data):
     for file_data in bundle_2['files']:
         file_data['path'] = file_data['path'].replace('.vcf', '.2.vcf')
     # WHEN adding them to the database
-    for bundle in [bundle_1, bundle_2]:
-        bundle_obj = store.add_bundle(bundle)
-        store.add_commit(bundle_obj)
+    to_store = []
+    with store.session.no_autoflush:
```

Comment:
It seems data gets flushed before we commit, resulting in 'autoflush errors'. So disabling autoflush for the duration of the adding